UpdateCTestConfiguration  from :/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/DartConfiguration.tcl
UpdateCTestConfiguration  from :/Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/DartConfiguration.tcl
Test project /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
Constructing a list of tests
Done constructing a list of tests
Updating test list for fixtures
Added 0 tests to meet fixture requirements
Checking test dependency graph...
Checking test dependency graph end
test 1
      Start  1: incorrect_solution_tests

1: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/incorrect_solution_tests
1: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
1: Test timeout computed to be: 5
1: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/_deps/googletest-src/googletest/src/gtest_main.cc
1: [==========] Running 6 tests from 2 test suites.
1: [----------] Global test environment set-up.
1: [----------] 5 tests from PacketTest
1: [ RUN      ] PacketTest.SerializeAndDeserializePositive
1: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:19: Failure
1: Expected equality of these values:
1:   deserializedPacket->magicNumber
1:     Which is: 43981
1:   0x1234
1:     Which is: 4660
1: 
1: [  FAILED  ] PacketTest.SerializeAndDeserializePositive (0 ms)
1: [ RUN      ] PacketTest.DeserializeInvalidData
1: [       OK ] PacketTest.DeserializeInvalidData (0 ms)
1: [ RUN      ] PacketTest.ZeroLengthPayload
1: [       OK ] PacketTest.ZeroLengthPayload (0 ms)
1: [ RUN      ] PacketTest.MaxValues
1: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:73: Failure
1: Expected equality of these values:
1:   deserializedPacket->magicNumber
1:     Which is: 43981
1:   0x1234
1:     Which is: 4660
1: 
1: [  FAILED  ] PacketTest.MaxValues (0 ms)
1: [ RUN      ] PacketTest.MinValues
1: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:96: Failure
1: Expected equality of these values:
1:   deserializedPacket->magicNumber
1:     Which is: 43981
1:   0x1234
1:     Which is: 4660
1: 
1: [  FAILED  ] PacketTest.MinValues (0 ms)
1: [----------] 5 tests from PacketTest (0 ms total)
1: 
1: [----------] 1 test from ProtocolTest
1: [ RUN      ] ProtocolTest.SendAndReceivePacket
1: Processing packet with sequence number 4321
1: [       OK ] ProtocolTest.SendAndReceivePacket (100 ms)
1: [----------] 1 test from ProtocolTest (100 ms total)
1: 
1: [----------] Global test environment tear-down
1: [==========] 6 tests from 2 test suites ran. (100 ms total)
1: [  PASSED  ] 3 tests.
1: [  FAILED  ] 3 tests, listed below:
1: [  FAILED  ] PacketTest.SerializeAndDeserializePositive
1: [  FAILED  ] PacketTest.MaxValues
1: [  FAILED  ] PacketTest.MinValues
1: 
1:  3 FAILED TESTS
 1/11 Test  #1: incorrect_solution_tests .........***Failed    0.43 sec
test 2
      Start  2: model1_tests

2: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/model1_tests
2: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
2: Test timeout computed to be: 5
2: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/_deps/googletest-src/googletest/src/gtest_main.cc
2: [==========] Running 6 tests from 2 test suites.
2: [----------] Global test environment set-up.
2: [----------] 5 tests from PacketTest
2: [ RUN      ] PacketTest.SerializeAndDeserializePositive
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:19: Failure
2: Expected equality of these values:
2:   deserializedPacket->magicNumber
2:     Which is: 43707
2:   0x1234
2:     Which is: 4660
2: 
2: [  FAILED  ] PacketTest.SerializeAndDeserializePositive (0 ms)
2: [ RUN      ] PacketTest.DeserializeInvalidData
2: [       OK ] PacketTest.DeserializeInvalidData (0 ms)
2: [ RUN      ] PacketTest.ZeroLengthPayload
2: [       OK ] PacketTest.ZeroLengthPayload (0 ms)
2: [ RUN      ] PacketTest.MaxValues
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:73: Failure
2: Expected equality of these values:
2:   deserializedPacket->magicNumber
2:     Which is: 43707
2:   0x1234
2:     Which is: 4660
2: 
2: [  FAILED  ] PacketTest.MaxValues (0 ms)
2: [ RUN      ] PacketTest.MinValues
2: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:96: Failure
2: Expected equality of these values:
2:   deserializedPacket->magicNumber
2:     Which is: 43707
2:   0x1234
2:     Which is: 4660
2: 
2: [  FAILED  ] PacketTest.MinValues (0 ms)
2: [----------] 5 tests from PacketTest (0 ms total)
2: 
2: [----------] 1 test from ProtocolTest
2: [ RUN      ] ProtocolTest.SendAndReceivePacket
2: Processing packet with sequence number 4321
2: [       OK ] ProtocolTest.SendAndReceivePacket (105 ms)
2: [----------] 1 test from ProtocolTest (105 ms total)
2: 
2: [----------] Global test environment tear-down
2: [==========] 6 tests from 2 test suites ran. (105 ms total)
2: [  PASSED  ] 3 tests.
2: [  FAILED  ] 3 tests, listed below:
2: [  FAILED  ] PacketTest.SerializeAndDeserializePositive
2: [  FAILED  ] PacketTest.MaxValues
2: [  FAILED  ] PacketTest.MinValues
2: 
2:  3 FAILED TESTS
 2/11 Test  #2: model1_tests .....................***Failed    0.43 sec
test 3
      Start  3: model10_tests

3: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/model10_tests
3: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
3: Test timeout computed to be: 5
3: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/_deps/googletest-src/googletest/src/gtest_main.cc
3: [==========] Running 6 tests from 2 test suites.
3: [----------] Global test environment set-up.
3: [----------] 5 tests from PacketTest
3: [ RUN      ] PacketTest.SerializeAndDeserializePositive
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:19: Failure
3: Expected equality of these values:
3:   deserializedPacket->magicNumber
3:     Which is: 0
3:   0x1234
3:     Which is: 4660
3: 
3: [  FAILED  ] PacketTest.SerializeAndDeserializePositive (0 ms)
3: [ RUN      ] PacketTest.DeserializeInvalidData
3: [       OK ] PacketTest.DeserializeInvalidData (0 ms)
3: [ RUN      ] PacketTest.ZeroLengthPayload
3: [       OK ] PacketTest.ZeroLengthPayload (0 ms)
3: [ RUN      ] PacketTest.MaxValues
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:73: Failure
3: Expected equality of these values:
3:   deserializedPacket->magicNumber
3:     Which is: 0
3:   0x1234
3:     Which is: 4660
3: 
3: [  FAILED  ] PacketTest.MaxValues (0 ms)
3: [ RUN      ] PacketTest.MinValues
3: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:96: Failure
3: Expected equality of these values:
3:   deserializedPacket->magicNumber
3:     Which is: 0
3:   0x1234
3:     Which is: 4660
3: 
3: [  FAILED  ] PacketTest.MinValues (0 ms)
3: [----------] 5 tests from PacketTest (0 ms total)
3: 
3: [----------] 1 test from ProtocolTest
3: [ RUN      ] ProtocolTest.SendAndReceivePacket
3: Processing packet with sequence number 4321
3: [       OK ] ProtocolTest.SendAndReceivePacket (105 ms)
3: [----------] 1 test from ProtocolTest (105 ms total)
3: 
3: [----------] Global test environment tear-down
3: [==========] 6 tests from 2 test suites ran. (105 ms total)
3: [  PASSED  ] 3 tests.
3: [  FAILED  ] 3 tests, listed below:
3: [  FAILED  ] PacketTest.SerializeAndDeserializePositive
3: [  FAILED  ] PacketTest.MaxValues
3: [  FAILED  ] PacketTest.MinValues
3: 
3:  3 FAILED TESTS
 3/11 Test  #3: model10_tests ....................***Failed    0.44 sec
test 4
      Start  4: model2_tests

4: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/model2_tests
4: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
4: Test timeout computed to be: 5
4: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/_deps/googletest-src/googletest/src/gtest_main.cc
4: [==========] Running 6 tests from 2 test suites.
4: [----------] Global test environment set-up.
4: [----------] 5 tests from PacketTest
4: [ RUN      ] PacketTest.SerializeAndDeserializePositive
4: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:19: Failure
4: Expected equality of these values:
4:   deserializedPacket->magicNumber
4:     Which is: 43981
4:   0x1234
4:     Which is: 4660
4: 
4: [  FAILED  ] PacketTest.SerializeAndDeserializePositive (0 ms)
4: [ RUN      ] PacketTest.DeserializeInvalidData
4: [       OK ] PacketTest.DeserializeInvalidData (0 ms)
4: [ RUN      ] PacketTest.ZeroLengthPayload
4: [       OK ] PacketTest.ZeroLengthPayload (0 ms)
4: [ RUN      ] PacketTest.MaxValues
4: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:73: Failure
4: Expected equality of these values:
4:   deserializedPacket->magicNumber
4:     Which is: 43981
4:   0x1234
4:     Which is: 4660
4: 
4: [  FAILED  ] PacketTest.MaxValues (0 ms)
4: [ RUN      ] PacketTest.MinValues
4: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:96: Failure
4: Expected equality of these values:
4:   deserializedPacket->magicNumber
4:     Which is: 43981
4:   0x1234
4:     Which is: 4660
4: 
4: [  FAILED  ] PacketTest.MinValues (0 ms)
4: [----------] 5 tests from PacketTest (0 ms total)
4: 
4: [----------] 1 test from ProtocolTest
4: [ RUN      ] ProtocolTest.SendAndReceivePacket
4: Processing packet with sequence number 4321
4: [       OK ] ProtocolTest.SendAndReceivePacket (105 ms)
4: [----------] 1 test from ProtocolTest (105 ms total)
4: 
4: [----------] Global test environment tear-down
4: [==========] 6 tests from 2 test suites ran. (105 ms total)
4: [  PASSED  ] 3 tests.
4: [  FAILED  ] 3 tests, listed below:
4: [  FAILED  ] PacketTest.SerializeAndDeserializePositive
4: [  FAILED  ] PacketTest.MaxValues
4: [  FAILED  ] PacketTest.MinValues
4: 
4:  3 FAILED TESTS
 4/11 Test  #4: model2_tests .....................***Failed    0.44 sec
test 5
      Start  5: model3_tests

5: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/model3_tests
5: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
5: Test timeout computed to be: 5
5: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/_deps/googletest-src/googletest/src/gtest_main.cc
5: [==========] Running 6 tests from 2 test suites.
5: [----------] Global test environment set-up.
5: [----------] 5 tests from PacketTest
5: [ RUN      ] PacketTest.SerializeAndDeserializePositive
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:19: Failure
5: Expected equality of these values:
5:   deserializedPacket->magicNumber
5:     Which is: 43981
5:   0x1234
5:     Which is: 4660
5: 
5: [  FAILED  ] PacketTest.SerializeAndDeserializePositive (0 ms)
5: [ RUN      ] PacketTest.DeserializeInvalidData
5: [       OK ] PacketTest.DeserializeInvalidData (0 ms)
5: [ RUN      ] PacketTest.ZeroLengthPayload
5: [       OK ] PacketTest.ZeroLengthPayload (0 ms)
5: [ RUN      ] PacketTest.MaxValues
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:73: Failure
5: Expected equality of these values:
5:   deserializedPacket->magicNumber
5:     Which is: 43981
5:   0x1234
5:     Which is: 4660
5: 
5: [  FAILED  ] PacketTest.MaxValues (0 ms)
5: [ RUN      ] PacketTest.MinValues
5: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:96: Failure
5: Expected equality of these values:
5:   deserializedPacket->magicNumber
5:     Which is: 43981
5:   0x1234
5:     Which is: 4660
5: 
5: [  FAILED  ] PacketTest.MinValues (0 ms)
5: [----------] 5 tests from PacketTest (0 ms total)
5: 
5: [----------] 1 test from ProtocolTest
5: [ RUN      ] ProtocolTest.SendAndReceivePacket
5: Processing packet with sequence number 4321
5: [       OK ] ProtocolTest.SendAndReceivePacket (105 ms)
5: [----------] 1 test from ProtocolTest (105 ms total)
5: 
5: [----------] Global test environment tear-down
5: [==========] 6 tests from 2 test suites ran. (105 ms total)
5: [  PASSED  ] 3 tests.
5: [  FAILED  ] 3 tests, listed below:
5: [  FAILED  ] PacketTest.SerializeAndDeserializePositive
5: [  FAILED  ] PacketTest.MaxValues
5: [  FAILED  ] PacketTest.MinValues
5: 
5:  3 FAILED TESTS
 5/11 Test  #5: model3_tests .....................***Failed    0.44 sec
test 6
      Start  6: model4_tests

6: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/model4_tests
6: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
6: Test timeout computed to be: 5
6: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/_deps/googletest-src/googletest/src/gtest_main.cc
6: [==========] Running 6 tests from 2 test suites.
6: [----------] Global test environment set-up.
6: [----------] 5 tests from PacketTest
6: [ RUN      ] PacketTest.SerializeAndDeserializePositive
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:19: Failure
6: Expected equality of these values:
6:   deserializedPacket->magicNumber
6:     Which is: 51966
6:   0x1234
6:     Which is: 4660
6: 
6: [  FAILED  ] PacketTest.SerializeAndDeserializePositive (0 ms)
6: [ RUN      ] PacketTest.DeserializeInvalidData
6: [       OK ] PacketTest.DeserializeInvalidData (0 ms)
6: [ RUN      ] PacketTest.ZeroLengthPayload
6: [       OK ] PacketTest.ZeroLengthPayload (0 ms)
6: [ RUN      ] PacketTest.MaxValues
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:73: Failure
6: Expected equality of these values:
6:   deserializedPacket->magicNumber
6:     Which is: 51966
6:   0x1234
6:     Which is: 4660
6: 
6: [  FAILED  ] PacketTest.MaxValues (0 ms)
6: [ RUN      ] PacketTest.MinValues
6: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:96: Failure
6: Expected equality of these values:
6:   deserializedPacket->magicNumber
6:     Which is: 51966
6:   0x1234
6:     Which is: 4660
6: 
6: [  FAILED  ] PacketTest.MinValues (0 ms)
6: [----------] 5 tests from PacketTest (0 ms total)
6: 
6: [----------] 1 test from ProtocolTest
6: [ RUN      ] ProtocolTest.SendAndReceivePacket
6: Processing packet with sequence number 4321
6: [       OK ] ProtocolTest.SendAndReceivePacket (105 ms)
6: [----------] 1 test from ProtocolTest (105 ms total)
6: 
6: [----------] Global test environment tear-down
6: [==========] 6 tests from 2 test suites ran. (105 ms total)
6: [  PASSED  ] 3 tests.
6: [  FAILED  ] 3 tests, listed below:
6: [  FAILED  ] PacketTest.SerializeAndDeserializePositive
6: [  FAILED  ] PacketTest.MaxValues
6: [  FAILED  ] PacketTest.MinValues
6: 
6:  3 FAILED TESTS
 6/11 Test  #6: model4_tests .....................***Failed    0.43 sec
test 7
      Start  7: model5_tests

7: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/model5_tests
7: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
7: Test timeout computed to be: 5
7: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/_deps/googletest-src/googletest/src/gtest_main.cc
7: [==========] Running 6 tests from 2 test suites.
7: [----------] Global test environment set-up.
7: [----------] 5 tests from PacketTest
7: [ RUN      ] PacketTest.SerializeAndDeserializePositive
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:19: Failure
7: Expected equality of these values:
7:   deserializedPacket->magicNumber
7:     Which is: 43981
7:   0x1234
7:     Which is: 4660
7: 
7: [  FAILED  ] PacketTest.SerializeAndDeserializePositive (0 ms)
7: [ RUN      ] PacketTest.DeserializeInvalidData
7: [       OK ] PacketTest.DeserializeInvalidData (0 ms)
7: [ RUN      ] PacketTest.ZeroLengthPayload
7: [       OK ] PacketTest.ZeroLengthPayload (0 ms)
7: [ RUN      ] PacketTest.MaxValues
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:73: Failure
7: Expected equality of these values:
7:   deserializedPacket->magicNumber
7:     Which is: 43981
7:   0x1234
7:     Which is: 4660
7: 
7: [  FAILED  ] PacketTest.MaxValues (0 ms)
7: [ RUN      ] PacketTest.MinValues
7: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/test.cpp:96: Failure
7: Expected equality of these values:
7:   deserializedPacket->magicNumber
7:     Which is: 43981
7:   0x1234
7:     Which is: 4660
7: 
7: [  FAILED  ] PacketTest.MinValues (0 ms)
7: [----------] 5 tests from PacketTest (0 ms total)
7: 
7: [----------] 1 test from ProtocolTest
7: [ RUN      ] ProtocolTest.SendAndReceivePacket
7: Processing packet with sequence number 4321
7: [       OK ] ProtocolTest.SendAndReceivePacket (105 ms)
7: [----------] 1 test from ProtocolTest (105 ms total)
7: 
7: [----------] Global test environment tear-down
7: [==========] 6 tests from 2 test suites ran. (105 ms total)
7: [  PASSED  ] 3 tests.
7: [  FAILED  ] 3 tests, listed below:
7: [  FAILED  ] PacketTest.SerializeAndDeserializePositive
7: [  FAILED  ] PacketTest.MaxValues
7: [  FAILED  ] PacketTest.MinValues
7: 
7:  3 FAILED TESTS
 7/11 Test  #7: model5_tests .....................***Failed    0.43 sec
test 8
      Start  8: model6_tests

8: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/model6_tests
8: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
8: Test timeout computed to be: 5
8: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/_deps/googletest-src/googletest/src/gtest_main.cc
8: [==========] Running 6 tests from 2 test suites.
8: [----------] Global test environment set-up.
8: [----------] 5 tests from PacketTest
8: [ RUN      ] PacketTest.SerializeAndDeserializePositive
8: [       OK ] PacketTest.SerializeAndDeserializePositive (0 ms)
8: [ RUN      ] PacketTest.DeserializeInvalidData
8: [       OK ] PacketTest.DeserializeInvalidData (0 ms)
8: [ RUN      ] PacketTest.ZeroLengthPayload
8: [       OK ] PacketTest.ZeroLengthPayload (0 ms)
8: [ RUN      ] PacketTest.MaxValues
8: [       OK ] PacketTest.MaxValues (0 ms)
8: [ RUN      ] PacketTest.MinValues
8: [       OK ] PacketTest.MinValues (0 ms)
8: [----------] 5 tests from PacketTest (0 ms total)
8: 
8: [----------] 1 test from ProtocolTest
8: [ RUN      ] ProtocolTest.SendAndReceivePacket
8: Processing packet with sequence number 4321
8: [       OK ] ProtocolTest.SendAndReceivePacket (103 ms)
8: [----------] 1 test from ProtocolTest (103 ms total)
8: 
8: [----------] Global test environment tear-down
8: [==========] 6 tests from 2 test suites ran. (103 ms total)
8: [  PASSED  ] 6 tests.
 8/11 Test  #8: model6_tests .....................   Passed    0.48 sec
test 9
      Start  9: model7_tests

9: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/model7_tests
9: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
9: Test timeout computed to be: 5
9: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/_deps/googletest-src/googletest/src/gtest_main.cc
9: [==========] Running 6 tests from 2 test suites.
9: [----------] Global test environment set-up.
9: [----------] 5 tests from PacketTest
9: [ RUN      ] PacketTest.SerializeAndDeserializePositive
9: [       OK ] PacketTest.SerializeAndDeserializePositive (0 ms)
9: [ RUN      ] PacketTest.DeserializeInvalidData
9: [       OK ] PacketTest.DeserializeInvalidData (0 ms)
9: [ RUN      ] PacketTest.ZeroLengthPayload
9: [       OK ] PacketTest.ZeroLengthPayload (0 ms)
9: [ RUN      ] PacketTest.MaxValues
9: [       OK ] PacketTest.MaxValues (0 ms)
9: [ RUN      ] PacketTest.MinValues
9: [       OK ] PacketTest.MinValues (0 ms)
9: [----------] 5 tests from PacketTest (0 ms total)
9: 
9: [----------] 1 test from ProtocolTest
9: [ RUN      ] ProtocolTest.SendAndReceivePacket
9: Processing packet with sequence number 4321
9: [       OK ] ProtocolTest.SendAndReceivePacket (103 ms)
9: [----------] 1 test from ProtocolTest (103 ms total)
9: 
9: [----------] Global test environment tear-down
9: [==========] 6 tests from 2 test suites ran. (103 ms total)
9: [  PASSED  ] 6 tests.
 9/11 Test  #9: model7_tests .....................   Passed    0.44 sec
test 10
      Start 10: model8_tests

10: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/model8_tests
10: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
10: Test timeout computed to be: 5
10: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/_deps/googletest-src/googletest/src/gtest_main.cc
10: [==========] Running 6 tests from 2 test suites.
10: [----------] Global test environment set-up.
10: [----------] 5 tests from PacketTest
10: [ RUN      ] PacketTest.SerializeAndDeserializePositive
10: [       OK ] PacketTest.SerializeAndDeserializePositive (0 ms)
10: [ RUN      ] PacketTest.DeserializeInvalidData
10: [       OK ] PacketTest.DeserializeInvalidData (0 ms)
10: [ RUN      ] PacketTest.ZeroLengthPayload
10: [       OK ] PacketTest.ZeroLengthPayload (0 ms)
10: [ RUN      ] PacketTest.MaxValues
10: [       OK ] PacketTest.MaxValues (0 ms)
10: [ RUN      ] PacketTest.MinValues
10: [       OK ] PacketTest.MinValues (0 ms)
10: [----------] 5 tests from PacketTest (0 ms total)
10: 
10: [----------] 1 test from ProtocolTest
10: [ RUN      ] ProtocolTest.SendAndReceivePacket
10: Processing packet with sequence number 4321
10: [       OK ] ProtocolTest.SendAndReceivePacket (105 ms)
10: [----------] 1 test from ProtocolTest (105 ms total)
10: 
10: [----------] Global test environment tear-down
10: [==========] 6 tests from 2 test suites ran. (105 ms total)
10: [  PASSED  ] 6 tests.
10/11 Test #10: model8_tests .....................   Passed    0.49 sec
test 11
      Start 11: model9_tests

11: Test command: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/model9_tests
11: Working Directory: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
11: Test timeout computed to be: 5
11: Running main() from /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/_deps/googletest-src/googletest/src/gtest_main.cc
11: [==========] Running 6 tests from 2 test suites.
11: [----------] Global test environment set-up.
11: [----------] 5 tests from PacketTest
11: [ RUN      ] PacketTest.SerializeAndDeserializePositive
11: [       OK ] PacketTest.SerializeAndDeserializePositive (0 ms)
11: [ RUN      ] PacketTest.DeserializeInvalidData
11: [       OK ] PacketTest.DeserializeInvalidData (0 ms)
11: [ RUN      ] PacketTest.ZeroLengthPayload
11: [       OK ] PacketTest.ZeroLengthPayload (0 ms)
11: [ RUN      ] PacketTest.MaxValues
11: [       OK ] PacketTest.MaxValues (0 ms)
11: [ RUN      ] PacketTest.MinValues
11: [       OK ] PacketTest.MinValues (0 ms)
11: [----------] 5 tests from PacketTest (0 ms total)
11: 
11: [----------] 1 test from ProtocolTest
11: [ RUN      ] ProtocolTest.SendAndReceivePacket
11: Processing packet with sequence number 4321
11: [       OK ] ProtocolTest.SendAndReceivePacket (103 ms)
11: [----------] 1 test from ProtocolTest (103 ms total)
11: 
11: [----------] Global tesErrors while running CTest
Output from these tests are in: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
t environment tear-down
11: [==========] 6 tests from 2 test suites ran. (104 ms total)
11: [  PASSED  ] 6 tests.
11/11 Test #11: model9_tests .....................   Passed    0.44 sec

36% tests passed, 7 tests failed out of 11

Total Test time (real) =   4.88 sec

The following tests FAILED:
	  1 - incorrect_solution_tests (Failed)
	  2 - model1_tests (Failed)
	  3 - model10_tests (Failed)
	  4 - model2_tests (Failed)
	  5 - model3_tests (Failed)
	  6 - model4_tests (Failed)
	  7 - model5_tests (Failed)
Test project /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build
      Start  1: incorrect_solution_tests
 1/11 Test  #1: incorrect_solution_tests .........***Failed    0.11 sec
      Start  2: model1_tests
 2/11 Test  #2: model1_tests .....................***Failed    0.12 sec
      Start  3: model10_tests
 3/11 Test  #3: model10_tests ....................***Failed    0.12 sec
      Start  4: model2_tests
 4/11 Test  #4: model2_tests .....................***Failed    0.11 sec
      Start  5: model3_tests
 5/11 Test  #5: model3_tests .....................***Failed    0.11 sec
      Start  6: model4_tests
 6/11 Test  #6: model4_tests .....................***Failed    0.12 sec
      Start  7: model5_tests
 7/11 Test  #7: model5_tests .....................***Failed    0.12 sec
      Start  8: model6_tests
 8/11 Test  #8: model6_tests .....................   Passed    0.12 sec
      Start  9: model7_tests
 9/11 Test  #9: model7_tests .....................   Passed    0.12 sec
      Start 10: model8_tests
10/11 Test #10: model8_tests .....................   Passed    0.11 sec
      Start 11: model9_tests
11/11 Test #11: model9_tests .....................   Passed    0.12 sec

36% tests passed, 7 tests failed out of 11

Total Test time (real) =   1.28 sec

The following tests FAILED:
	  1 - incorrect_solution_tests (Failed)
	  2 - model1_tests (Failed)
	  3 - model10_tests (Failed)
	  4 - model2_tests (Failed)
	  5 - model3_tests (Failed)
	  6 - model4_tests (Failed)
	  7 - model5_tests (Failed)
Errors while running CTest
Output from these tests are in: /Users/ittehad/Documents/Coding/Python/Turing/Tester_cpp_codebench/tasks/473654/build/Testing/Temporary/LastTest.log
Use "--rerun-failed --output-on-failure" to re-run the failed cases verbosely.
